Dockerfile

```diff
@@ -0,0 +1,34 @@
+
+FROM node:8.16 as builder
+
+# Create build directory
+WORKDIR /tmp/build
+
+# Install first dependencies, so docker can cache this layer
+COPY package*.json truffle-config.js ./
+COPY contracts contracts
+
+# Then the rest of files
+COPY . .
+
+RUN npm install
```

Comment:
Let's use yarn

Reply:
Yes. for sure, but don't I have to `npm install yarn` first?